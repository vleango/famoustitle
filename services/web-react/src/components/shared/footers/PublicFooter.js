import React from 'react';

import './css/public-footer.css';

export class PublicFooter extends React.Component {

    render() {
        return (
            <footer className="bd-footer text-muted">
                <div className="container">
                    <ul className="bd-footer-links">
                        <li><a rel="noopener noreferrer" target="_blank" href="https://github.com/vleango/famoustitle">GitHub</a></li>
                        <li><a rel="noopener noreferrer" target="_blank" href="https://stackoverflow.com/users/1316386/tha-leang">Stack Overflow</a></li>
                        <li><a rel="noopener noreferrer" target="_blank" href="https://www.linkedin.com/in/tha-leang-1b340959">LinkedIn</a></li>
                    </ul>
                    <p>Created by Tha Leang</p>
                </div>
            </footer>
        );
    }
}
