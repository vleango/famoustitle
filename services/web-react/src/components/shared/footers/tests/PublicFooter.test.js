import React from 'react';
import { shallow } from 'enzyme';

import { PublicFooter } from '../PublicFooter';

describe('Shared', () => {
    describe('Footers', () => {
        describe('PublicFooter', () => {
            let wrapper = shallow(<PublicFooter />);

            it('should correctly render PublicFooter', () => {
                expect(wrapper).toMatchSnapshot();
            });
        });

    });
});
