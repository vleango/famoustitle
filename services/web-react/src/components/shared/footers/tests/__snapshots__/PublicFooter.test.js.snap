// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Shared Footers PublicFooter should correctly render PublicFooter 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <PublicFooter />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div
        className="container"
>
        <ul
                className="bd-footer-links"
        >
                <li>
                        <a
                                href="https://github.com/vleango/famoustitle"
                                rel="noopener noreferrer"
                                target="_blank"
                        >
                                GitHub
                        </a>
                </li>
                <li>
                        <a
                                href="https://stackoverflow.com/users/1316386/tha-leang"
                                rel="noopener noreferrer"
                                target="_blank"
                        >
                                Stack Overflow
                        </a>
                </li>
                <li>
                        <a
                                href="https://www.linkedin.com/in/tha-leang-1b340959"
                                rel="noopener noreferrer"
                                target="_blank"
                        >
                                LinkedIn
                        </a>
                </li>
        </ul>
        <p>
                Created by Tha Leang
        </p>
</div>,
      "className": "bd-footer text-muted",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <ul
            className="bd-footer-links"
>
            <li>
                        <a
                                    href="https://github.com/vleango/famoustitle"
                                    rel="noopener noreferrer"
                                    target="_blank"
                        >
                                    GitHub
                        </a>
            </li>
            <li>
                        <a
                                    href="https://stackoverflow.com/users/1316386/tha-leang"
                                    rel="noopener noreferrer"
                                    target="_blank"
                        >
                                    Stack Overflow
                        </a>
            </li>
            <li>
                        <a
                                    href="https://www.linkedin.com/in/tha-leang-1b340959"
                                    rel="noopener noreferrer"
                                    target="_blank"
                        >
                                    LinkedIn
                        </a>
            </li>
</ul>,
          <p>
            Created by Tha Leang
</p>,
        ],
        "className": "container",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <li>
                <a
                                href="https://github.com/vleango/famoustitle"
                                rel="noopener noreferrer"
                                target="_blank"
                >
                                GitHub
                </a>
</li>,
              <li>
                <a
                                href="https://stackoverflow.com/users/1316386/tha-leang"
                                rel="noopener noreferrer"
                                target="_blank"
                >
                                Stack Overflow
                </a>
</li>,
              <li>
                <a
                                href="https://www.linkedin.com/in/tha-leang-1b340959"
                                rel="noopener noreferrer"
                                target="_blank"
                >
                                LinkedIn
                </a>
</li>,
            ],
            "className": "bd-footer-links",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <a
                  href="https://github.com/vleango/famoustitle"
                  rel="noopener noreferrer"
                  target="_blank"
>
                  GitHub
</a>,
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "GitHub",
                  "href": "https://github.com/vleango/famoustitle",
                  "rel": "noopener noreferrer",
                  "target": "_blank",
                },
                "ref": null,
                "rendered": "GitHub",
                "type": "a",
              },
              "type": "li",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <a
                  href="https://stackoverflow.com/users/1316386/tha-leang"
                  rel="noopener noreferrer"
                  target="_blank"
>
                  Stack Overflow
</a>,
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Stack Overflow",
                  "href": "https://stackoverflow.com/users/1316386/tha-leang",
                  "rel": "noopener noreferrer",
                  "target": "_blank",
                },
                "ref": null,
                "rendered": "Stack Overflow",
                "type": "a",
              },
              "type": "li",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <a
                  href="https://www.linkedin.com/in/tha-leang-1b340959"
                  rel="noopener noreferrer"
                  target="_blank"
>
                  LinkedIn
</a>,
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "LinkedIn",
                  "href": "https://www.linkedin.com/in/tha-leang-1b340959",
                  "rel": "noopener noreferrer",
                  "target": "_blank",
                },
                "ref": null,
                "rendered": "LinkedIn",
                "type": "a",
              },
              "type": "li",
            },
          ],
          "type": "ul",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Created by Tha Leang",
          },
          "ref": null,
          "rendered": "Created by Tha Leang",
          "type": "p",
        },
      ],
      "type": "div",
    },
    "type": "footer",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="container"
>
          <ul
                    className="bd-footer-links"
          >
                    <li>
                              <a
                                        href="https://github.com/vleango/famoustitle"
                                        rel="noopener noreferrer"
                                        target="_blank"
                              >
                                        GitHub
                              </a>
                    </li>
                    <li>
                              <a
                                        href="https://stackoverflow.com/users/1316386/tha-leang"
                                        rel="noopener noreferrer"
                                        target="_blank"
                              >
                                        Stack Overflow
                              </a>
                    </li>
                    <li>
                              <a
                                        href="https://www.linkedin.com/in/tha-leang-1b340959"
                                        rel="noopener noreferrer"
                                        target="_blank"
                              >
                                        LinkedIn
                              </a>
                    </li>
          </ul>
          <p>
                    Created by Tha Leang
          </p>
</div>,
        "className": "bd-footer text-muted",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <ul
              className="bd-footer-links"
>
              <li>
                            <a
                                          href="https://github.com/vleango/famoustitle"
                                          rel="noopener noreferrer"
                                          target="_blank"
                            >
                                          GitHub
                            </a>
              </li>
              <li>
                            <a
                                          href="https://stackoverflow.com/users/1316386/tha-leang"
                                          rel="noopener noreferrer"
                                          target="_blank"
                            >
                                          Stack Overflow
                            </a>
              </li>
              <li>
                            <a
                                          href="https://www.linkedin.com/in/tha-leang-1b340959"
                                          rel="noopener noreferrer"
                                          target="_blank"
                            >
                                          LinkedIn
                            </a>
              </li>
</ul>,
            <p>
              Created by Tha Leang
</p>,
          ],
          "className": "container",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <li>
                  <a
                                    href="https://github.com/vleango/famoustitle"
                                    rel="noopener noreferrer"
                                    target="_blank"
                  >
                                    GitHub
                  </a>
</li>,
                <li>
                  <a
                                    href="https://stackoverflow.com/users/1316386/tha-leang"
                                    rel="noopener noreferrer"
                                    target="_blank"
                  >
                                    Stack Overflow
                  </a>
</li>,
                <li>
                  <a
                                    href="https://www.linkedin.com/in/tha-leang-1b340959"
                                    rel="noopener noreferrer"
                                    target="_blank"
                  >
                                    LinkedIn
                  </a>
</li>,
              ],
              "className": "bd-footer-links",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <a
                    href="https://github.com/vleango/famoustitle"
                    rel="noopener noreferrer"
                    target="_blank"
>
                    GitHub
</a>,
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "GitHub",
                    "href": "https://github.com/vleango/famoustitle",
                    "rel": "noopener noreferrer",
                    "target": "_blank",
                  },
                  "ref": null,
                  "rendered": "GitHub",
                  "type": "a",
                },
                "type": "li",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <a
                    href="https://stackoverflow.com/users/1316386/tha-leang"
                    rel="noopener noreferrer"
                    target="_blank"
>
                    Stack Overflow
</a>,
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Stack Overflow",
                    "href": "https://stackoverflow.com/users/1316386/tha-leang",
                    "rel": "noopener noreferrer",
                    "target": "_blank",
                  },
                  "ref": null,
                  "rendered": "Stack Overflow",
                  "type": "a",
                },
                "type": "li",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <a
                    href="https://www.linkedin.com/in/tha-leang-1b340959"
                    rel="noopener noreferrer"
                    target="_blank"
>
                    LinkedIn
</a>,
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "LinkedIn",
                    "href": "https://www.linkedin.com/in/tha-leang-1b340959",
                    "rel": "noopener noreferrer",
                    "target": "_blank",
                  },
                  "ref": null,
                  "rendered": "LinkedIn",
                  "type": "a",
                },
                "type": "li",
              },
            ],
            "type": "ul",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Created by Tha Leang",
            },
            "ref": null,
            "rendered": "Created by Tha Leang",
            "type": "p",
          },
        ],
        "type": "div",
      },
      "type": "footer",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
